config/config.example.json

```diff
@@ -63,7 +65,8 @@
     "responders": [
       {
         "type": "team",
-        "id": "a uuid value"
+        "id": "a uuid value",
```

Comment:
I discovered a while back that this is actually optional/this array can be empty. In these cases OpsGenie will use the OpsGenie API key to determine the responders (the key belongs to an integration which is associated with a team).

Reply:
Yep, the only issue right now is that the `OpsGenieConfig` types in `operations-utilities` don't allow these to be optional :sweat_smile:.

I'll fix them here once I get a chance to fix them in that repo :smile: